libs/domains/services/data-access/src/lib/domains-services-data-access.ts

```diff
@@ -153,8 +166,12 @@ export type Helm = _Helm & {
   // @deprecated Prefer use `service_type` from API instead of `serviceType`
   serviceType: HelmType
 }
+export type Terraform = _Terraform & {
+  // @deprecated Prefer use `service_type` from API instead of `serviceType`
+  serviceType: TerraformType
```

Comment:
You can probably remove it no? or have you seen some error without?

Reply:
<img width="419" height="63" alt="Screenshot 2025-08-04 at 15 38 22" src="https://github.com/user-attachments/assets/168ed1fc-4ef3-4d01-a5c8-98dbc79e6574" />

____   
  
Yeah, a few ones 😅 
171 just by removing this line 😅   

But actually that's a fair point, we need to refactor this at some point 🙂